editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/tests/preferences/StagingRepositoriesPreferencePageIT.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.projecttemplates.tests.preferences;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.jface.preference.PreferenceDialog;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.dialogs.PreferencesUtil;
+import org.fusesource.ide.projecttemplates.internal.ProjectTemplatesActivator;
+import org.fusesource.ide.projecttemplates.preferences.StagingRepositoriesConstants;
+import org.fusesource.ide.projecttemplates.preferences.StagingRepositoriesPreferencePage;
+import org.fusesource.ide.projecttemplates.preferences.StagingRepositoryDialog;
+import org.fusesource.ide.projecttemplates.preferences.initializer.StagingRepositoriesPreferenceInitializer;
+import org.junit.Test;
+
+/**
+ * @author brianf
+ *
+ */
+public class StagingRepositoriesPreferencePageIT {
+	
+	private String testList = "name1,http://my.url1;name2,http://my.url2";
+	private IPreferenceStore preferenceStore;
+
+	@Test
+	public void testStagingRepositoryPreferencePage() throws Exception {
+		// grab the preference store
+		preferenceStore = ProjectTemplatesActivator.getDefault().getPreferenceStore();
+		assertThat(preferenceStore).isNotNull();
+		
+		// set new test values
+		preferenceStore.setValue(StagingRepositoriesConstants.ENABLE_STAGING_REPOSITORIES, true);
+		preferenceStore.setValue(StagingRepositoriesConstants.STAGING_REPOSITORIES, testList);
+
+		// make sure they set properly
+		assertThat(preferenceStore.getBoolean(StagingRepositoriesConstants.ENABLE_STAGING_REPOSITORIES) == true);
```

Comment:
assertThat(preferenceStore.getBoolean(StagingRepositoriesConstants.ENABLE_STAGING_REPOSITORIES)).isTrue()

SonarLint plugin is reporting this kind of issue